README.md

```diff
@@ -159,6 +159,42 @@ docker-compose -f docker/app.yml up
 
 In case of Oracle database, see [official documentation](https://github.com/oracle/docker-images/blob/master/OracleDatabase/SingleInstance/README.md)
 
+
+## Using database migrations
+
+If you had alredy run the application before creating your first migration, some database tables might be alredy 
+created automatically on application startup (see EnsureCreated() method on DatabaseStartup.cs file for more details).
```

Comment:
Replace method EnsureCreated by Migrate()

Reply:
I just replaced it, instructing the user to replace EnsureCreated by Migrate() if the wish to run migrations on application startup.

Reply:
Just for clarification, I dit not change the template files of DatabaseStartup.cs file itself, just put the info on the docs because for it to work correctly the migrations would need to be created first (maybe automatically created by the generator)